tests/features/steps/payment_steps.py

```diff
@@ -74,9 +97,28 @@ def step_impl(context, url):
 def step_impl(context, message):
     assert message in context.resp.data
 
+
 @then('I should not see "{message}"')
 def step_impl(context, message):
-	assert message not in context.resp.data
+    assert message not in context.resp.data
+
+
+@then('I should be returned nothing')
+def step_impl(context):
+    assert context.resp.status_code == status.HTTP_204_NO_CONTENT
+    assert context.resp.data == ''
+
+
+@then('I should see a {http_code} response')
+def step_impl(context, http_code):
+    desired_http_code = getattr(status, http_code)
+    assert context.resp.status_code == desired_http_code
+
+
+@then('I should be returned a {http_code} response')
+def step_impl(context, http_code):
+    desired_http_code = getattr(status, http_code)
+    assert context.resp.status_code == desired_http_code
 
```

Comment:
They look very similar, but they correspond to two different `then` statements.  This might be confusing though.  If I can re-use the then statement, that'd be best.